src/main/java/core/basesyntax/service/impl/SupplyFruitOperation.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitStorage;
+import core.basesyntax.service.FruitOperation;
+import java.util.Map;
+
+public class SupplyFruitOperation implements FruitOperation {
+    private Map<String, Fruit> storage = FruitStorage.getFruitStorage();
```

Comment:
we don't need this field in this class. Let's communicate  with storage only through StrorageService